.github/workflows/frontend.yml

```
@@ -62,3 +62,39 @@ jobs:
       - name: Frontend Tests
         run: npm run ci:frontend:test
         working-directory: simulator-ui
+  e2e:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Install dependencies
+        run: npm ci --cache .npm --force
+        working-directory: simulator-ui
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+        working-directory: simulator-ui
+      - name: Run Playwright tests
+        run: |
+          echo "Current directory is:"
+          pwd
+          ls
+          npm run e2e
+          echo "------------------------------are reports existent?-------------------------"
+          ls
+        working-directory: simulator-ui
+        continue-on-error: true
+      - uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
+        with:
+          name: test-results
+          path: test-results/
```

the above jobs have the `working-directory: simulator-ui` line, but these jobs here cannot. therefore the "working directory" must be added:
```suggestion
          path: simulator-ui/test-results/
```

---

Yes you're right, I didn't know these where separate jobs. Thanks!